recipe/meta.yaml

```diff
@@ -34,12 +34,10 @@ requirements:
     - r-base
     - libgomp                    # [linux]
     - llvm-openmp                # [osx]
-    - zlib                       # [not win]
-    - {{ native }}zlib           # [win]
+    - zlib
   run:
     - r-base
-    - {{ native }}gcc-libs       # [win]
-    - {{ native }}zlib           # [win]
+    - zlib           # [win]
```

Comment:
```suggestion
```

Reply:
Ah, sorry - I missed that and I had already ticked the automerge! I'll try to remember to clean it later.